Lasse/LittleWindows.asm

```diff
@@ -61,41 +66,31 @@
 ; DefWindowProcA 	ebp+58			(user32.dll)
 ; SetBkMode 		ebp+5c			(gdi32.dll)
 
-; Global variables and structures
-; hInstance 		ebp+60 
-; CommandLine 		ebp+64
+; Global variables
+; hInstance             ebp+60 
+; CommandLine           ebp+64
 
 ; How to build
-; ml /coff LittleWindows.asm
+; ml /coff LittleWindows.asm -link /subsystem:windows 
 ;  or, for example. if you need to specify include path to windows.inc:
-; ml /coff /I c:\masm32\include LittleWindows.asm
+; ml /coff LittleWindows.asm /I c:\masm32\include -link /subsystem:windows 

```

Comment:
I think these need to be updated to align with the build instructions in the PR description?